### PR TITLE
แปลภาษาไทย

### PR DESCRIPTION
**Pull request by TeamSTNETRadio**

### CODE REVIEW

File: plus/unsubscribe/unsubscribe.md
Comment (sorasukt) [inline plus/unsubscribe/unsubscribe.md:1]: I think 🤔 file name is different from **official ver**

File: SUMMARY.md
Comment (sorasukt) [inline SUMMARY.md:1]: # Change Header name
1. Seconds header change to เริ่มใช้งาน Plus+ กันเถอะ

File: plus/unsubscribe/unsubscribe.md
Comment (sorasukt) [reply plus/unsubscribe/unsubscribe.md:1]: Files name `plus/unsubscribe/README.md` is not `plus/unsubscribe/unsubscribe.md`

**Review by sorasukt — CHANGES_REQUESTED** [review]
## Please update following:

- [ ] **Files name**: Make sure all files are same name like [support](https://github.com/STNETRadio/support)
- [x] **Thai Lag change**: Make sure all transactions are correct ✅

**Review by sorasukt — CHANGES_REQUESTED** [review]
Same review from last review

**Comment by TeamSTNETRadio** [comment]
Now, we’re updating this commit @sorasukt

**Review by sorasukt — APPROVED** [review]
#1 Is verified

**Pull request merged**